raw_request.go

```diff
@@ -18,10 +17,11 @@ type apiResponse struct {
 	Result      json.RawMessage `json:"result,omitempty"`
 	Description string          `json:"description,omitempty"`
 	ErrorCode   int             `json:"error_code,omitempty"`
+	Parameters  struct {
+		RetryAfter int `json:"retry_after"`
+	} `json:"parameters,omitempty"`
```

Comment:
I guess, we should add the `migrate_to_chat_id` field according the specification